src/app/App.js

```diff
@@ -75,11 +75,11 @@ function App() {
 				/>
 				<Route
 					path='ptsignup'
-					element={<Navigate to='//forms.gle/hHqKT4tX6W1n8VHJ9' />}
+					element={<Navigate to='//forms.gle/2uHLu1FdWZbGyYeW8' />}
 				/>
 				<Route
 					path='ptsignups'
-					element={<Navigate to='//forms.gle/hHqKT4tX6W1n8VHJ9' />}
+					element={<Navigate to='//forms.gle/2uHLu1FdWZbGyYeW8' />}
```

Comment:
Nitpick [non-blocking]: seems we are repeating the PT application URL several times. It might be cleaner to declare and export a constant from `PT.js`.

Reply:
Yeah this is a great idea. Might make this an intern task.